mobile/native/src/api.rs

```diff
@@ -217,3 +217,8 @@ pub fn get_seed_phrase() -> Result<SyncReturn<Vec<String>>> {
     let seed_phrase = ln_dlc::get_seed_phrase()?;
     Ok(SyncReturn(seed_phrase))
 }
+
+pub fn register_beta(email: String) -> Result<()> {
+    tracing::debug!("Registering beta user: {}", email);
```

Comment:
Nothing is happening in here?

Reply:
yup, the goal was to put the frontend part, till the API is this PR (as it was described). 
could have left a TODO to make it more clear in code though.